java/src/org/openqa/selenium/remote/service/DriverService.java

```diff
@@ -63,6 +64,7 @@
 public class DriverService implements Closeable {
 
   private static final String NAME = "Driver Service Executor";
+  private static final Logger LOG = Logger.getLogger(SeleniumManager.class.getName());
```

Comment:
Since this `LOG` is in the `DriverService` class, the class name in this line should be `DriverService` and not `SeleniumManager`.

Reply:
copy paste strikes again. thanks.